setup.py

```diff
@@ -38,8 +38,10 @@ def read_module_contents():
     install_requires=[
         'Flask',
         'Flask-Restful',
+        'Flask-SQLAlchemy',
+        'SQLAlchemy',
         'koji',
-        'pygresql',
+        'psycopg2',
```

Comment:
This should be probably `psycopg2-binary`, otherwise I'm getting following deprecation warning when running live tests.

```
venv/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from bin
ary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
```

Reply:
Fixed